tests/Signals_perftest.m

```diff
@@ -152,7 +155,21 @@ function test_SignalOps(testCase, SignalOps)
             for i = 1:testCase.Reps
               post(testCase.A, s);
             end
-          end         
+          end
+        case {'delay'}
+          % special case: have to use `runSchedule`     
+          while (testCase.keepMeasuring)
+            for i = 1:testCase.Reps
+              testCase.Net.runSchedule();
+            end
+          end
+        case {'flatten'}
```

Comment:
You don't need a special case here, right?  Remove this case entirely and change the above to this:
```
        case {'flatten'}
          s = testCase.Net.subscriptableOrigin('s'); % create a subscriptable signal
          s.field = testCase.A;
          s_flat = feval(SignalOps, s.field);
      end
```
or even...
```
        case {'flatten'}
          testCase.B.post(testCase.A);
          b_flat = feval(SignalOps, testCase.B);
      end
```

Reply:
Hm, I'm not sure either of these do what we want to accomplish. Correct me if I'm wrong, but this is how I see it:

For your first example, you want to measure the actual "flattening" of a signal field, `s.field` in a subscriptable signal, `s`. To do so, you want to measure the direct assignment of a signal, `testCase.A` to `s.field`, *after* posting to `testCase.A`, which your first example doesn't do. 

Your second example would make sense if `flatten` had the same behavior as `flattenStruct`, as we talked about yesterday. However, since it doesn't, when posting to `testCase.A`, `b_flat` doesn't get updated. 

In short, I don't see how you can updated a flattened signal without assigning to a signal field of a subscriptable signal, *after* that signal field has first been updated. Maybe you see another way?

Reply:
To use the first example the lines need to be switched, yes:
```
        case {'flatten'}
          s = testCase.Net.subscriptableOrigin('s'); % create a subscriptable signal
          s_flat = feval(SignalOps, s.field);
          s.field = testCase.A;
      end
```
Both of these work and measure what we're interested in.  I would prefer the second as it's one fewer line of code.

Reply:
Oh right your second example works if you flip the two middle lines, so that you post to `testCase.B` after creating `b_flat`

Reply:
You still need to remove the flatten case, let it run the code in the `otherwise` block.  Namely just post a value into A, which will the propagate to b_flat.

Reply:
when I make dumb mistakes on a re-request like forgetting to take things out or leaving things in you can feel free to make the change yourself if you want. and to do the squash and merge in github.